symfony/webpack-encore-bundle/1.9/config/packages/webpack_encore.yaml

```diff
@@ -7,7 +7,12 @@ webpack_encore:
     # Set attributes that will be rendered on all script and link tags
     script_attributes:
         defer: true
+        # Uncomment (also under link_attributes) if using Turbo Drive
+        # https://turbo.hotwire.dev/handbook/drive#reloading-when-assets-change
+        # 'data-turbo-track': reload
```

Comment:
why not uncomment by default?

Reply:
Feels weird to sprinkle this attribute for everyone, including on non-Turbo projects